kritis-charts/README.md

```diff
@@ -1,5 +1,34 @@
 # kritis
 
+## Setting up Kritis
+
+### Creating a Kubernetes Cluster
+
+To run the kritis admission webhook, you will need access to a version 1.9.2+ Kubernetes cluster.
+You can create one by running
+```
+gcloud container clusters create <CLUSTER NAME> \
+--cluster-version 1.9.7-gke.3 \
+--zone us-central1-a \
+--num-nodes 1
+```
+
+### Enabling the Container Analysis API
+You will need to enable the Container Analysis API and enable Vulnerability Scanning in your Google Cloud Console project.
+Instructions can be found in the `Before you Begin` section of the [Getting Image Vulnerabilities](https://cloud.google.com/container-registry/docs/get-image-vulnerabilities#before_you_begin) docs.
+kritis can only inspect images hosted in projects that have both of these enabled.
+
+### Creating a Container Analysis Secret
+You will need to create a Kubernetes secret, which will provide kritis with the auth required to get vulnerability information for images.
+To create the secret:
+1. Create a service account with `Container Analysis Admin` permissions in the Google Cloud Console project that hosts the images kritis will be inspecting
+2. Download a JSON key for the service account
+3. Rename the key to `kritis.json`
+4. Create a Kubernetes secret by running:
+```
+kubectl create secret generic <SECRET NAME> --from-file=<path to kritis.json>
+```
+
 ## Installing Kritis
```

Comment:
Can you copy the command from kritis-charts/ here and update to use helm certgen plugin

Reply:
done